### ISSUE TITLE
[Feature Request] Support for "offline" mode

### ISSUE THREAD

**Issue by ivdimitro** [joBr99/nspanel-lovelace-ui#360]
### FEATURE DESCRIPTION
When there is no connection to the appdaemon back end the display should show the screensaver (or similar screen) with time and date taken from tasmota and indication that the back end is not available. 

### ADDITIONAL CONTEXT
Updating the time via MQTT creates excessive network traffic and reduce the reliability of the display. By reading the time from tasmota those downsides can be mitigated and increase the "wife acceptance factor" :)

### PANEL / FIRMWARE VERION
EU

**Comment by joBr99** [comment]
this has way to many disadvantages

- breaks timeAdditonalTemplate
- breaks timeFormat
- requires configuration on multipliple places (tasmota script and apps.yaml)
- requires correct timezone on tasmota

also it isn't possible to detect if appdaemon is still alive, this would require heartbeat messages which are creating the same or more traffic than setting the time over mqtt

you can create a tasmota rule in the berry script that is sending the time every minute like blakadder did for the stock ui

https://github.com/blakadder/nspanel/blob/01db07ac52109204bb1df09535e96e40669d43c1/nspanel.be#L434
https://github.com/blakadder/nspanel/blob/01db07ac52109204bb1df09535e96e40669d43c1/nspanel.be#L151

you just need to send the payload "time~XXXXX" every minute via the nextion.send command


just keep your homeassistant online and your wife will be happy

**Comment by ivdimitro** [comment]
Yeah, it really looks more complicated than I though. Thanks for the clarification.
